README.md

```diff
@@ -1,4 +1,5 @@
-# makectl
+# makectl    ![MIT license](https://img.shields.io/badge/License-MIT-blue.svg)     ![made-with-bash](https://img.shields.io/badge/Made%20with-Bash-1f425f.svg)    ![Documentation Status](https://readthedocs.org/projects/ansicolortags/badge/?version=latest)    ![PRs Welcome](https://img.shields.io/badge/PRs-welcome-brightgreen.svg?style=flat-square)     ![GitHub issues](https://img.shields.io/github/issues/Naereen/StrapDown.js.svg)
```

Comment:
actually this project is going to have

- Github actions badge as the example found in #2 
- Documentation badge pointing to future docs.rs page
- Crates.io badge

Reply:
ok I removed the old ones and added 3 new ones. It's ready for review.